packages/expo-notifications/src/Notifications.types.ts

```diff
@@ -252,7 +252,53 @@ export type ChannelAwareTriggerInput = {
   channelId: string;
 };
 
-// @docsMissing
+/**
+ * Schedulable trigger inputs (that are not a plain Date value or time value)
```

Comment:
```suggestion
 * Schedulable trigger inputs (that are not a plain date value or time value)
```

If date is a syntax here, then we should use inline code syntax highlights to represent it.